testData/solutions/linux.json5

```diff
@@ -1533,5 +1533,57 @@
                 "type": "gpii.deviceReporter.alwaysInstalled"
             }
         ]
+    },
+    "net.gpii.async": {
```

Comment:
I think this is more elaborate than we need. I put the "explode" solution into the registry because it can be used live to demonstrate some user-visible functionality, that is, whether particular error messages appear. Given that this async handler is only useful for tests, I suggest that we don't implement all the machinery of a real solution but simply use the existing mock settings handler at https://github.com/GPII/universal/blob/master/gpii/node_modules/testing/src/Mocks.js#L284 with just a small improvement to stick the delay in the options. We can then just populate the mock registry at https://github.com/GPII/universal/blob/master/gpii/node_modules/testing/src/Mocks.js#L420 with an appropriate mock for a settings handler which is already in use in the acceptance tests.